.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-mqcloud
```

Comment:
this needs to match the value in ibm_catalog.json which is `deploy-arch-ibm-mq-cloud`. This is the value we set up in catalog so needs to match here